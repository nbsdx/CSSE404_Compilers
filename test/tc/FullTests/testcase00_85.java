class fred
{
	public static void main(String[] arg)
	{
		boolean x = false;
		int y = 0;
		y = 0;
		x = y;
	}
}
