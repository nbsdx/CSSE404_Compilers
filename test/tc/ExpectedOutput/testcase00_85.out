Cannot assign type int to variable x of type boolean
